mongodbatlas/resource_mongodbatlas_project_api_key.go

```diff
@@ -244,7 +244,7 @@ func resourceMongoDBAtlasProjectAPIKeyUpdate(ctx context.Context, d *schema.Reso
 
 	if d.HasChange("description") {
 		newDescription := d.Get("description").(string)
-		if _, _, err := connV2.ProgrammaticAPIKeysApi.UpdateApiKeyRoles(ctx, projectID, apiKeyID, &admin.CreateAtlasProjectApiKey{
+		if _, _, err := connV2.ProgrammaticAPIKeysApi.UpdateApiKeyRoles(ctx, projectID, apiKeyID, &admin.UpdateAtlasProjectApiKey{
```

Comment:
is this change wanted? we don't have the create `CreateAtlasProjectApiKey` function anymore?

Reply:
Just a naming change that is aligned with the operation. The structure of `admin.CreateAtlasProjectApiKey` and `admin.UpdateAtlasProjectApiKey` are identical.